src/components/Button/index.tsx

```diff
@@ -7,11 +7,37 @@ import {
 
 import { scrollIntoView } from "../../utils/scrollIntoView"
 
+export const checkIsSecondary = (props: {
+  variant?: string
+  isSecondary?: boolean
+}) => {
+  const { variant, isSecondary } = props
+  // These two variants do not have secondary styling, so prevent overrides
+  return {
+    "data-secondary":
+      !["solid", "link"].includes(variant || "solid") && isSecondary,
+  }
+}
+
```

Comment:
This allows use of `data-secondary` under both the `Button` and `IconButton` components, and avoiding the need to do forced styling in the theme config (i.e. add `!important` flag to a style value, say)

Reply:
Works perfectly 💪🏼